conans/test/integration/toolchains/gnu/test_autotoolstoolchain.py

```diff
@@ -276,3 +279,41 @@ def build(self):
     })
     client.run("export tool")
     client.run("create consumer -pr:h host -pr:b build --build=missing")
+
+
+@pytest.mark.skipif(platform.system() != "Darwin", reason="requires Xcode")
```

Comment:
Let's add a profile host so we can run this test on all the platforms, for instance:

```python
profile_host = textwrap.dedent("""
[settings]
arch=x86_64
build_type=Release
compiler=apple-clang
compiler.cppstd=gnu17
compiler.libcxx=libc++
compiler.version=15
os=Macos
""")
```
Then, the conan command will look like:
```python
client.run("install . --profile:build=profile_build --profile:host=profile_host")
```
Applying those changes, we won't depend on the OS anymore:

```suggestion
```